src/Tribe/Timezones.php

```diff
@@ -131,6 +131,14 @@ public static function event_end_timestamp( $event_id, $timezone = null ) {
 	 * @return int
 	 */
 	protected static function get_event_timestamp( $event_id, $type = 'Start', $timezone = null ) {
+		static $timestamps = [];
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: 9cfd502cfc3ac8032ed42b589e329e2e</sub>

Reply:
Ignoring.